darwin/future/helpers/exception_handler.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import sys
+from typing import List, Optional, Union
+
+from rich.console import Console
+
+
```

Comment:
Added a _potential_ exception handler, after working out how to discover whether the lib is being used from CLI or SDK.